server/src/test/java/org/elasticsearch/cluster/coordination/ClusterFormationFailureHelperTests.java

```diff
@@ -121,10 +121,10 @@ public void testScheduling() {
                     deterministicTaskQueue.advanceTime();
                 }
             }
+            assertThat(warningCount.get(), is(1L));
+            assertThat(deterministicTaskQueue.getCurrentTimeMillis() - startTimeMillis, is(expectedDelayMillis));
+            mockLogAppender.assertAllExpectationsMatched();
```

Comment:
do we need this method to be called explicitly?
perhaps this should be added to inside the MockLogAppender::capturing lambda/close?